pipeline/testing/formal/trigger.py

```
@@ -0,0 +1,188 @@
+        raise ValueError("checklist 无 items: %s" % checklist_path)
+    results = [evaluate_item(meta, item) for item in items]
+    fill_trace = build_fill_trace(meta, items)
+    risk_field = (doc.get("risk_gate") or {}).get("field", "risk_level")
+    risk_level, risk_present = get_field(meta, risk_field)
+
+    positives = [r for r in results if r["kind"] == "positive" and r["matched"]]
+    negatives = [r for r in results if r["kind"] == "negative" and r["matched"]]
+
+    if not risk_present:
+        final, reason = "needs_risk_level", (
+            "fail-closed：元数据缺少 %s（机械字段），无法完成风险门（AD 不允许默认放行）" % risk_field
```

Ruff (0.16.2)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)